packages/auth/test/integration/webdriver/redirect.test.ts

```diff
@@ -70,6 +75,26 @@ browserDescribe('WebDriver redirect IdP test', driver => {
     expect(await driver.call(RedirectFunction.REDIRECT_RESULT)).to.be.null;
   });
 
+  // Redirect works with middleware for now
+  xit('is blocked by middleware', async () => {
```

Comment:
Is this temporarily being skipped because it's not implemented yet?

Reply:
Yep, I'll switch it to `it` once that part is working